src/php/server_fpm.rs

```diff
@@ -11,6 +11,8 @@ use crate::{php::structs::PhpServerSapi};
 use crate::utils::network::find_available_port;
 use std::process::Child;
 
+use wsl::is_wsl;
```

Comment:
```suggestion
#[cfg(not(target_family = "windows"))]
use wsl::is_wsl;
```

Sorry, forgot this one: since FPM is only available on unix-like system, the statement triggers a warning on native windows 😅

Reply:
@Pierstoval Out of interest to better understand Rust: this means you import the method only on non-windows systems, right? What happens on windows with the `&& !is_wsl()` call? How does that compile?

Reply:
It compiles because the function is compiled differently in Windows, because FPM does not exist on windows 😁(as of the annotations used in this file)

Reply:
Oh right, I had a look at the file again in my IDE and then it became obvious. Checking the PR alone was not enough to get the full view. Thanks!

Reply:
Yep, I think that's the biggest problem with "conditionnal compilation": you sometimes don't get the whole picture in a PR 😕